formula-android/src/test/java/com/instacart/formula/android/ActivityStoreContextTest.kt

```diff
@@ -57,6 +57,25 @@ class ActivityStoreContextTest {
             .assertValues(false, true)
     }
 
+    @Test fun `is fragment started is a behavior relay - new subscription gets latest emission first`() {
```

Comment:
Behavior relay is an RxJava concept; probably the test should be named `isFragmentStarted emits initial value first`